lib/jwt_sessions/authorization.rb

```diff
@@ -48,6 +48,10 @@ def authorize_refresh_by_access_header!
       cookieless_auth(:access)
     end
 
+    def decode_access_tokens
+      @_raw_token = token_from_headers(:access, required: false) || token_from_cookies(:access, required: false)
+    end
```

Comment:
could you please provide a README entry explaining how this method is expected to be used